evalml/automl/automl_search.py

```diff
@@ -1632,10 +1700,94 @@ def full_rankings(self):
             "search_order",
             pd.Series(self._results["search_order"]),
         )  # place search_order after pipeline_name
-        rankings_df.sort_values("ranking_score", ascending=ascending, inplace=True)
+
+        ranking_column = "ranking_score"
+        if self.use_recommendation:
```

Comment:
Is there any benefit to modifying the API of `full_rankings()` to have the ranking column be selectable via the function call rather than the AutoML __init__ setting?  Probably not, I guess.  Wouldn't want the user to be able to select "recommendation_score" with `self.use_recommendation` not having been set...